docs/docs-content/vertex/system-management/account-management/credentials.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_label: "User Credentials"
+title: "User Credentials"
+description: "Update and manage the user credentials"
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 20
+tags: ["vertex", "management", "account", "credentials"]
+keywords: ["self-hosted", "vertex"]
+---
+
+You can manage the credentials of the admin user by logging in to the system console. You can also enable passwordless login for the admin user by adding passkeys. 
+
+Use the sections below to learn how to manage user credentials, such as changing the admin password, and enabling passwordless login by adding passkeys.
+
+## Change Password
+
+Use the following steps to change the password of the admin user.
+
+### Prerequisites
+
+- Access to the Palette VerteX system console.
+
+- Current password of the admin user.
+
+### Steps
+
+1. Log in to the Palette VerteX system console. Refer to [Access the System Console](../system-management.md#access-the-system-console) guide.
+
+2. From the **left Main Menu** select **My Account**.
+
+3. In the field **Current Password** type the current password of the admin user.
+
+4. In the field **New Password** type the new password for the admin user.
+
+5. Type the new password again in the field **Re-Enter New Password**.
+
+6. Click **Change Password** to save the changes.
+
+
+### Validate 
+
+1. Log out of the system console. You can log out by clicking the **Logout** button in the bottom right corner of the **left Main Menu**.
+
+2. Log in to the system console. Refer to [Access the System Console](../system-management.md#access-the-system-console) guide.
+
+2. Use the new email address and your current password to log in to the system console.
+
+A successful login indicates that the password has been changed successfully.
+
+
+## Passkeys
+
+VerteX supports passwordless login (using the [WebAuthn Level2 protocol](https://developers.yubico.com/WebAuthn/Concepts/WebAuthn_Level_2_Features_and_Enhancements.html)) for the admin user through passkeys. You can also use a physical device, such as a YubiKey, to log in to the system console. If you enable the passkeys, we recommend you create at least two passkeys for the admin user, in case you lose one of the passkeys.
```

Comment:
```suggestion
VerteX supports passwordless login (using the [WebAuthn Level2 protocol](https://developers.yubico.com/WebAuthn/Concepts/WebAuthn_Level_2_Features_and_Enhancements.html)) for the admin user through passkeys. You can also use a physical device, such as a YubiKey, to log in to the system console. If you enable the passkeys, we recommend you create at least two passkeys for the admin user.
```

Reply:
Repetition with the caution below.

Reply:
This is intentional, as we really want to drive home the point. Some will skim and miss this, but the bright color callout will catch everyone's attention.